migration_steps/prepare_source_data/counts_verification/sql/count_existing_pc1.sql

```diff
@@ -0,0 +1,299 @@
+DROP SCHEMA IF EXISTS countverification CASCADE;
```

Comment:
Do we need both the CLIENT-PILOT-ONE counts and the overall counts? Are we interested in the unfiltered counts?

Should the filename by cp1 rather than pc1?

Reply:
how strange - never noticed the filename typo! Will Amend

The aim with this ticket is to track CP1 data only. We can expand on this to count other data in subsequent tickets, such as IN-1066